presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraMetadata.java

```diff
@@ -132,7 +131,7 @@ private ConnectorTableMetadata getTableMetadata(ConnectorSession session, Schema
         ImmutableList.Builder<SchemaTableName> tableNames = ImmutableList.builder();
         for (String schemaName : listSchemas(session, schemaNameOrNull)) {
             try {
-                for (String tableName : schemaProvider.getAllTables(schemaName)) {
+                for (String tableName : cassandraSession.getCaseSensitiveTableNames(schemaName)) {
```

Comment:
`SchemaTableName` constructor does that. I don't want to check for duplicates here for the same reason as for keyspaces.